python/acl_anthology/collections/types.py

```diff
@@ -25,6 +35,22 @@ class PaperDeletionType(Enum):
     """Paper was removed.  A removal occurs in rare circumstances where serious ethical or legal issues arise, such as plagiarism."""
 
 
+class PaperType(Enum):
+    """Type of paper.
+
+    Currently only exists to support a few paper instances that are marked up as 'backmatter'.
+    """
+
+    PAPER = "paper"
+    """A regular paper."""
+
+    FRONTMATTER = "frontmatter"
+    """The frontmatter of a volume."""
+
+    BACKMATTER = "backmatter"
```

Comment:
Do we ever use this? It's not formally encoded.

Reply:
http://aclanthology.org/venues/tc/ uses this, and one instance in EarlyMT. 13 backmatter instances in total.

Reply:
We might want to formally encode the possible options in the schema, come to think of it, as it currently allows arbitrary strings.